kolibri/plugins/facility/assets/src/views/UserPage/index.vue

```diff
@@ -72,55 +127,81 @@
           @changeSort="changeSortHandler"
         >
           <template #header="{ header, colIndex }">
-            <span :class="{ visuallyhidden: colIndex === 5 }">{{ header.label }}</span>
-            <span v-if="colIndex === 2">
-              <CoreInfoIcon
-                class="tooltip"
-                :iconAriaLabel="coreString('identifierAriaLabel')"
-                :tooltipText="coreString('identifierTooltip')"
-              />
-            </span>
+            <template v-if="colIndex === 0">
+              <KCheckbox
+                :label="selectAllLabel$()"
+                :checked="selectAllState.checked"
+                :indeterminate="selectAllState.indeterminate"
+                :showLabel="false"
+                :ariaLabel="selectAllLabel$()"
+                @change="handleSelectAllToggle"
+              >
+                <span class="visuallyhidden">{{ selectAllLabel$() }}</span>
+              </KCheckbox>
+            </template>
+            <template v-else>
+              <span :class="{ visuallyhidden: colIndex === 7 }">{{ header.label }}</span>
+              <span v-if="colIndex === 3">
+                <CoreInfoIcon
+                  class="tooltip"
+                  :iconAriaLabel="coreString('identifierAriaLabel')"
+                  :tooltipText="coreString('identifierTooltip')"
+                />
+              </span>
+            </template>
           </template>
+
           <template #cell="{ content, colIndex, row }">
             <span v-if="colIndex === 0">
+              <KCheckbox
```

Comment:
We do need either a visually-hidden label or an aria-label (please follow whichever @radinamatic indicates is the preferred option based on the question above) for these checkboxes

> Each row checkbox should announce its label by screen readers: e.g. “Select John Doe, coach.”